files/image_config/procdockerstats/procdockerstats

```diff
@@ -0,0 +1,211 @@
+# !/usr/bin/env python
+'''
+This code is for a specific daemon, process-docker.
+Which sends process and docker CPU/memory utilization data to DB every 2 mins.
```

Comment:
"process-docker" is not the name of the daemon. Suggest changing this comment as follows:

```
'''
procdockerstats

Daemon which periodically gathers process and docker statistics and pushes the data to STATE_DB
'''
```